src/azure-cli-core/HISTORY.rst

```diff
@@ -33,10 +37,6 @@ Release History
 ++++++
 * Minor fixes
 
-2.0.36
-++++++
-* Minor fixes
-
```

Comment:
why you remove it? My original comment was about removing redundant entries in your PR

Reply:
oops.. i misunderstood your comment, adding it back